app/controllers/ReturnsController.scala

```diff
@@ -152,6 +152,7 @@ class ReturnsController @Inject()(val messagesApi: MessagesApi,
       showReturnsBreadcrumb = isReturnsPageRequest,
       currentYear = dateService.now().getYear,
       hasFlatRateScheme = customerDetail.fold(false)(_.hasFlatRateScheme),
+      isOptOutMtdVatUser = false,
```

Comment:
Is there a reason this is hardcoded? It should be determined by the following logic (meaning you'll need to retrieve the mandation status)
```if (feature switch is on && mandation status == NonMTDfB) true else false```

Which can be simplified to:
```isOptOutMtdVatUser = feature switch is on && mandation status == NonMTDfB ```